src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -211,22 +210,26 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
               >
                 <div />
               </div>
-              <div
-                className={`nav-party-logo ${
-                  isEventScheduleVisible && "clicked"
-                }`}
-                onClick={toggleEventSchedule}
-              >
-                {navbarTitle} Schedule
-              </div>
-              <VenuePartygoers />
+              {venue && (
+                <>
+                  <div
+                    className={classNames("nav-party-logo", {
+                      clicked: isEventScheduleVisible,
+                    })}
+                    onClick={toggleEventSchedule}
+                  >
+                    {navbarTitle} Schedule
+                  </div>
+                  <VenuePartygoers />
+                </>
+              )}
```

Comment:
It depends if we want to hack this quickly, or do it properly. The better fix here would be to pull out the 'non-generic' parts of the NavBar into their own separate components.

The 'generic navbar' would then have various 'slots' that can be used to render children. On the non-admin side of things, these 'requires a venue' parts would then be passed into one of these 'slots'.

This is the sort of pattern that I expect we will be using more and more of as we clean up/genericise our components.

Reply:
(for what it's worth, I don't think it would take very much extra effort to do the proper 'slots' fix)

Reply:
Skimming through this, there are actually quite a lot of parts that won't be needed in the admin panel, so we could avoid all of the 'mess' of the hooks/state/etc related to those things by separating them into the 'user-side only' navbar, that then 'injects' the various specific parts into the generic NavBar's 'slots'.

So I think that would be my preferred approach here.

Reply:
What's your idea of handling these so called 'slots' something like a boolean?

Reply:
Are these slots supposed to be passed as children, render prop or something else?

Reply:
Please elaborate on this change because I need this for my admin v2/admin v3 UI.

Reply:
Children would make sense if there was only 1 ‘slot’ required, but I imagine we’ll want multiple, so probably render props/similar.

Reply:
Then exact specifics I’m not sure if off the top of my head, but as a basic example, there might be the ‘right hand side tools’ slot that would add the various things (like radio/etc) on that side of the navbar. And then perhaps there is a separate ‘left hand side slot’ that has things on that side.

Perhaps when we get the ‘hamburger menu’ there is a slot that lets us add additional items to that (beyond the default items), etc.

Reply:
I believe @mike-lvov may have similar ideas around using render props/similar to clean up/better isolate areas of our code, so he may be able to share some thoughts/insight on this too.

Reply:
render props sounds like a good approach. I would expect it to have 3 slots. right, center, left. Each of them should be able to accept either an array of elements or just one element. Aligned with `flex`.

Reply:
@denisdimitrov I thought you said that you had reworked this to use render props? Yet that doesn't seem to be the case here?